### PR TITLE
3.8.fp12: Add bug fixes release note

### PR DESCRIPTION
**Pull request by jbremond**

### CODE REVIEW

File: changelog.rst [hunk 1]
```diff
@@ -21,6 +21,22 @@ Enhancements
 * Ability to view the list of packages for images generated from a software component.
 * Ability to generate a Docker image from a Linux appliance without the kernel.
 
+Bug Fixes
+~~~~~~~~~
+* 13489 Fix incompatible version error when generating Oracle 7 img
```
Comment (emuus) [inline changelog.rst]: img -> image

File: changelog.rst [hunk 2]
```diff
@@ -21,6 +21,22 @@ Enhancements
...
+* 13551 Make all azure agent packages sticky
```
Comment (emuus) [inline changelog.rst]: Azure (capital)

File: changelog.rst [hunk 3]
```diff
@@ -21,6 +21,22 @@ Enhancements
...
+* 13350 UForge system update: `yum update docker` breaks docker command
+* 10912 Empty folders in MySoftware doesn't exist in generated machine
```
Comment (emuus) [inline changelog.rst]: doesn't exist -> are not present

File: changelog.rst [hunk 4]
```diff
@@ -21,6 +21,22 @@ Enhancements
...
+* 12518 Fix ssh connection for openstack
+* 13291 Slash in MySoftware files not well handled
+* 13302 Some appliances having an interactive profile do not have the warning icon when composing a blueprint
```
Comment (emuus) [inline changelog.rst]: having -> with

**Comment by jbremond** [comment]
Thank you @emuus for your comments. Please find a new version

**Pull request merged**